pkg/reconciler/common/proxy.go

```diff
@@ -81,24 +79,6 @@ func ApplyProxySettings(u *unstructured.Unstructured) error {
 	return nil
 }
 
-// InjectLabelOnNamespace will add a label on tekton-pipelines and
-// openshift-pipelines namespace to disable proxy in these namespace
-func InjectLabelOnNamespace() mf.Transformer {
-	return func(u *unstructured.Unstructured) error {
-		kind := strings.ToLower(u.GetKind())
-		if kind != "namespace" {
-			return nil
-		}
-		name := u.GetName()
-		if name == "tekton-pipelines" || name == "openshift-pipelines" {
-			labels := u.GetLabels()
-			labels["operator.tekton.dev/disable-proxy"] = "true"
```

Comment:
Side note to @piyush-garg, are we also skipping `openshift-*` pods for the proxy ?